domains/server.linusx.json

```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "linuzzx",
+           "email": "",
+           "discord": "531070363795456000",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.QWYtxCl8R04xwSTqLOAPnFuG9cA703fkvIQQDdsxKElbKV1cXNJHTvfJtR04mbg4f7JOfnMqD1vtkGY1EbbZepR6Z-cGgLZMWENHML-2dvzja-lqnDQz2OhQjjMaGzla17PJcQJRnkJ-bbr0vd4HIXT9vq3lMr14RZ1UBV_-LFOqeI8q7vqFPDdVcOd5ziOQ2pB2BwSh1YKbjhyDhdVJ3hxGAfEdO3Q9fm954RMMO-iSW7gGuqUbBC5OaRb0RwyCClKcZnVortxue2AGL2eRMNd2EGK5sv6ozk83KcLPzpinRdTGaTOEFFvjl0JVbfITRYRGcfteOe2e4jrxArAF6w.CEz69C4n7ICEIE6UdxzOhw.S6FNria37m7qHUGAlhwm11HhjPAcIy1fD7nXaA0oQZskk1_EeoKsHMeUm0o-WI3MvTgTSe7PbMA33HyU6cNvJ4CslRR-bODWav3U1r1ZVhToWZ2Ai2m7koaV7EnOdD1C.VvfnuJj2e8gYw7_-f5i-qw"
+        },
+    
+        "record": {
+            "A": ["192.168.2.166"]
```

Comment:
That is a private IP

Reply:
doesn't it work if I only want to access it locally?

Reply:
No, It won't work. For domains you need a public IP, not a local one.